presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveBasicTableStatistics.java

```diff
@@ -126,10 +126,8 @@ public void testCreatePartitioned()
                 "WHERE n_nationkey <> 23", tableName));
 
         try {
-            if (getHiveVersionMajor() < 3) {
-                BasicStatistics tableStatistics = getBasicStatisticsForTable(onHive(), tableName);
-                assertThatStatisticsAreNotPresent(tableStatistics);
-            }
+            BasicStatistics tableStatistics = getBasicStatisticsForTable(onHive(), tableName);
+            assertThatStatisticsAreNotPresent(tableStatistics);
```

Comment:
Add a comment why this assertion matters, so that someone doesn't put an `if` around this again (like i did).
E.g. link to the issue.

Reply:
Also, can you add a test for CREATE external table? 
(partitioned and not partitioned)